airbyte-integrations/connectors/source-sendgrid/source_sendgrid/sendgrid.yaml

```diff
@@ -247,7 +248,7 @@ streams:
         $ref: "*ref(definitions.requester)"
         request_options_provider:
           request_parameters:
-            limit: 1000
+            limit: "1000"
```

Comment:
The alternative to this, is we change the type of `RequestInput` from `RequestInput = Union[str, Mapping[str, str]]` to `RequestInput = Union[str, Mapping[str, Any]]`. That would then make 1000 a valid input